### PR TITLE
Fix issue 115 "Cannot read property 'type' of undefined"

### PR DESCRIPTION
**Pull request by JuanMaRuiz**
Add new condition to `opportunityRules` method to avoid failure when a rules does not have details property

fixes: #115

### CODE REVIEW

File: lib/output.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ const opportunities = lighthouseResult => {
 
   const opportunityRules = rules.filter(rule => {
     const {details} = audits[rule.id];
-    return details.type === 'opportunity' && details.overallSavingsMs > 0;
+    return Boolean(details) && details.type === 'opportunity' && details.overallSavingsMs > 0;
```
Comment (exterkamp) [inline lib/output.js:62]: I'm a double ! guy, but this wfm 👍

**Review by exterkamp — APPROVED** [review]
Restarting Travis b/c it failed on a 429.

**Comment by JuanMaRuiz** [comment]
> Restarting Travis b/c it failed on a 429.

It seems that now is working properly.

**Comment by edm00se** [comment]
The merge commit of this into master appears to have failed for both travis ci build tasks.

**Pull request merged**